inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/file/FileSource.java

```diff
@@ -48,9 +48,6 @@ public class FileSource extends StreamSource {
     @ApiModelProperty("Agent Cluster tag")
     private String clusterTag;
```

Comment:
Do we need this field?

Reply:
Might not be used for now but the agent cluster do have a cluster name AND  cluster tag, right?

Reply:
On second though this is indeed redundant, the group has already specified the clusterTag. Sources belonging to some group need not specify cluster tags again.